src/subdomains/analytics/application/mappers/coin-tracking-history-dto.mapper.ts

```diff
@@ -68,7 +69,11 @@ export class CoinTrackingHistoryDtoMapper {
         sellAsset: null,
         fee: c.feePercent != 0 ? (c.targetAmount * c.feePercent) / (1 - c.feePercent) : null,
         feeAsset: c.feePercent != 0 ? this.getAssetSymbolCT(c.targetAsset) : null,
-        exchange: 'LOCK.space Staking',
+        exchange: c.isReinvest
+          ? 'LOCK.space Staking'
+          : c.targetAddress.address === c.staking.withdrawalAddress.address
```

Comment:
Use `c.targetAddress.isEqual(c.staking.withdrawalAddress)` for all address comparisons.